chapters/package-information.md

```diff
@@ -295,7 +295,7 @@ The metadata for the package download location field is shown in Table 19.
 | --------- | ----- |
 | Required | Yes |
 | Cardinality | 1..1 |
-| Format | Uniform resource locator \| VCS location \| `NONE` \| `NOASSERTION`<br><br>For version-controlled files, a VCS location is similar to a URL and has the following syntax:<br><br>`<vcs_tool>+<transport>://<host_name>[/<path_to_repository>][@<revision_tag_or_branch>][#<sub_path>]`<br><br>This VCS location compact notation (inspired and mostly adopted from [pip][pip-vcs] as of 2015-02-20) supports referencing locations in version control systems such as [Git][], [Mercurial][], [Subversion][] and [Bazaar][], and specifies the type of VCS tool using url prefixes: `git+`, `hg+`, `bzr+`, `svn+` and specific transport schemes such as SSH or HTTPS.<br><br>Specifying sub-paths, branch names, a commit hash, a revision or a tag name is recommended, and supported using the `@` delimiter for commits and the `#` delimiter for sub-paths.<br><br>Using user names and password in the `<host_name>` is not supported and should be considered as an error. User access control to URLs or VCS repositories shall be handled outside of an SPDX document.<br><br>In VCS location compact notations, the trailing slashes in `<host_name>`, `<path_to_repository>` are not significant. Leading and trailing slashes in `<sub_path>` are not significant. |
+| Format | Uniform resource locator \| VCS location \| `NONE` \| `NOASSERTION`<br><br>For version-controlled files, a VCS location is similar to a URL and has the following syntax:<br><br>`<vcs_tool>+<transport>://<host_name>[/<path_to_repository>][@<revision_tag_or_branch>][#<sub_path>]`,<br><br>where `@<revision_tag_or_branch>` is either a git tag or a git branch. It is highly recommended to use a full path like `refs/heads/<branch>` or `refs/tags/<tag>`, rather than only `<branch>` or `<tag>`, in order to prevent issues caused by shadowing (see [gitrevisions](https://mirrors.edge.kernel.org/pub/software/scm/git/docs/gitrevisions.html) manual page). <br><br>This VCS location compact notation (inspired and mostly adopted from [pip][pip-vcs] as of 2015-02-20) supports referencing locations in version control systems such as [Git][], [Mercurial][], [Subversion][] and [Bazaar][], and specifies the type of VCS tool using url prefixes: `git+`, `hg+`, `bzr+`, `svn+` and specific transport schemes such as SSH or HTTPS.<br><br>Specifying sub-paths, branch names, a commit hash, a revision or a tag name is recommended, and supported using the `@` delimiter for commits and the `#` delimiter for sub-paths.<br><br>Using user names and password in the `<host_name>` is not supported and should be considered as an error. User access control to URLs or VCS repositories shall be handled outside of an SPDX document.<br><br>In VCS location compact notations, the trailing slashes in `<host_name>`, `<path_to_repository>` are not significant. Leading and trailing slashes in `<sub_path>` are not significant. |
```

Comment:
Suggestion:

For git, `<revision_tag_or_branch>` is a `<refname>` or `<sha1>` as described by [gitrevisions(1)](https://mirrors.edge.kernel.org/pub/software/scm/git/docs/gitrevisions.html). It is RECOMMENDED to use the unambiguous `refs/heads/<branch>` or `refs/tags/<tag>`, rather than only `<branch>` or `<tag>`, in order to prevent issues caused by shadowing.

Reply:
Thank you, I've updated with your proposal.